### PR TITLE
Change from gatekeeper to kyverno

### PR DESCRIPTION
**Pull request by corang**

### CODE REVIEW

File: kustomizations/bigbang/common/values-bigbang.yaml [hunk 1]
```diff
@@ -214,6 +151,155 @@ eckoperator:
...
 
+kyverno:
+  enabled: true
+  values:
+    replicaCount: 5
+    resources:
+      limits:
+        cpu: "2"
+      requests:
+        cpu: "1"
```
Comment (RothAndrew) [inline kustomizations/bigbang/common/values-bigbang.yaml:162]: Missing memory request and limit

File: kustomizations/bigbang/common/values-bigbang.yaml [hunk 2]
```diff
@@ -214,6 +151,155 @@ eckoperator:
...
+          any:
+          - resources:
+              namespaces:
+              - "zarf"
+      disallow-privilege-escalation:
+        exclude:
+          any:
+          - resources:
+              names:
+              - "acid*"
```
Comment (RothAndrew) [inline kustomizations/bigbang/common/values-bigbang.yaml:179]: Is this needed?

**Review by RothAndrew — CHANGES_REQUESTED** [review]

File: kustomizations/bigbang/common/values-bigbang.yaml (hunk 1 above)
Comment (corang) [reply kustomizations/bigbang/common/values-bigbang.yaml:163]: added

File: kustomizations/bigbang/common/values-bigbang.yaml (hunk 2 above)
Comment (corang) [reply kustomizations/bigbang/common/values-bigbang.yaml]: testing without it, I think it is but that was pretty early in playing with kyverno that I added it

Comment (corang) [reply kustomizations/bigbang/common/values-bigbang.yaml]: Yeah, its not working anymore, I'll dive into exactly why I needed it/why the acid containers are privileged and see if there's a better way

Comment (corang) [reply kustomizations/bigbang/common/values-bigbang.yaml:191]: made the exclusion more specific

**Review by RothAndrew — APPROVED** [review]

**Pull request merged**